fa25-team-a/eda/Merge_Resale_Data.ipynb

```
@@ -0,0 +1,536 @@
+        "output_dir = Path('/content')  # directory to save output\n",
+        "\n",
+        "touchable_file = data_dir / 'touchable_resale_final_round.csv'\n",
+        "unique_file = data_dir / 'unique_to_dataset1.csv'\n",
+        "parsed_file = data_dir / 'ParsedResale_updatedSept2025.csv'\n",
+        "duplicates_output_file = output_dir / 'duplicate_property_keys_review.csv'\n",
+        "combined_output_file = output_dir / 'touchable_resale_final_round_combined_reconciled.csv'\n",
+        "\n",
+        "# --- Load datasets ---\n",
+        "df_touchable = pd.read_csv(touchable_file)\n",
+        "df_unique = pd.read_csv(unique_file)\n",
+        "df_parsed = pd.read_csv(parsed_file)\n",
```

Ruff (0.14.1)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)